proto/info/info.proto

```diff
@@ -1366,3 +1369,20 @@ message ZInfoKubeCluster {
   // Virtual Machines running on the cluster
   repeated KubeVMIInfo eve_vm_apps = 6;
 }
+
+message DimmRankInfo {
+  string rank_name = 1;
+  int64 ce_count = 2;
+  int64 ue_count = 3;
+}
+
+message ECCMemoryControllerInfo {
+  string controller_name = 1;
+  int64 ce_count = 2;
+  int64 ue_count = 3;
+  repeated DimmRankInfo ranks = 4;
+}
```

Comment:
Please document what these fields are. rank, ce, ue, etc.

Is the repeated DimRankInfo one entry per dimm? Or for something else?

Reply:
For the DimRankInfo we have one entry per dimm.